### DISCUSSION TITLE
What is a best coding style

### DISCUSSION THREAD

**Discussion by librarylai** [RexHung0302/Hexschool-web-layout-training#56] [Q&A] [answered]
Do you think, What is a best coding style??

**Comment by RexHung0302** [comment, marked as answer]
I believe is shit code style - https://github.com/doggy8088/state-of-the-art-shitcode/blob/zh-tw/README.zh-TW.md?fbclid=IwAR26si5A1FzqNCy-2ajECInLXWKq5zcXHU_RhWaaJAUy1ylZYx_vTi-d32g&fs=e&s=cl
